account_move_name_sequence/readme/DESCRIPTION.rst

```diff
@@ -7,6 +7,75 @@ In Odoo version 14.0, the number of journal entries can be manually set by the u
 * as it is error-prone, they added a *Resequence* wizard to re-generate the journal entry numbers, which can be considered as illegal in many countries,
 * the `piece of code <https://github.com/odoo/odoo/blob/14.0/addons/account/models/sequence_mixin.py>`_ that handle this is not easy to understand and quite difficult to debug.
 
+Readme de complemento para:
+https://github.com/OCA/account-financial-tools/tree/14.0/account_move_name_sequence
+
+In Odoo version 13.0 and previous versions, the number of journal entries was generated from a sequence configured on the journal.
+
+In Odoo version 14.0, the number of journal entries can be manually set by the user. Then, the number attributed for next journal entries in the same journal is computed by a complex piece of code that guesses the format of the journal entry number from the number of the journal entry which was manually entered by the user. It has several drawbacks:
+
+* the available options for the sequence are limited,
+* it is not possible to configure the sequence in advance before the deployment in production,
+* as it is error-prone, they added a *Resequence* wizard to re-generate the journal entry numbers, which can be considered as illegal in many countries,
+* the `piece of code <https://github.com/odoo/odoo/blob/14.0/addons/account/models/sequence_mixin.py>`_ that handle this is not easy to understand and quite difficult to debug.
+
+Odoo>=v14.0 raises new concurrency issues since it locks the last journal entry of the journal to get the new number causing a bottleneck
+Even if you only are creating a draft journal entry it locks the last one
+It applies to all accounting Journal Entries
+
+e.g.
+
+ - Customer Invoices,
+ - Credit Notes
```

Comment:
Only first item has a trailing comma, either remove it or add a comma to all of them